cross-chain/arbitrum/deploy_l2/21_authorize_wormhole_gateway_as_minter.ts

```diff
@@ -0,0 +1,25 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { execute } = deployments
+  const { governance } = await getNamedAccounts()
+
+  const ArbitrumWormholeGateway = await deployments.get(
+    "ArbitrumWormholeGateway"
+  )
+
+  await execute(
+    "ArbitrumTBTC",
+    { from: governance, log: true, waitConfirmations: 1 },
```

Comment:
Ah, this is why it works. Well, this will work only for Goerli with governance address set to the one we control. This will not work on mainnet because the dev (deployment) team does not own the `governance` address.

Reply:
Ok, so do we want to authorize the Wormhole Gateway first before transferring the ownership to the governance? Or should we leave any authorization entirely to the governance?

Reply:
We need to authorize the gateway so that it can be used. Let's just change the order of steps to do not require any action from the council.

Reply:
Done [c4a8155](https://github.com/keep-network/tbtc-v2/pull/554/commits/c4a815513410da8091c51a9c534636fb12c63577)